DietPi-Software | Kodi

Joulinar opened the pull request:
At the moment download of Kodi desktop item will fail if no desktop was selected because the target desktop icon folder did not exist

**Status**: Ready
- [x] update `dietpi-software`

**Reference**: https://dietpi.com/phpbb/viewtopic.php?t=8995

**Commit list/description**:
- DietPi-Software | Kodi: check for the availability of the desktop icon folder before going to download the Kodi desktop icon

Joulinar committed:
v7.2

DietPi-Software | Kodi: check for the availability of the desktop icon folder before going to download the Kodi desktop icon

MichaIng commented:
Many thanks. Indeed Kodi can be installed without desktop, even without X server in case of Odroid N2 + C4. So now the desktop icon is skipped when no desktop is installed. Just to throw in the an alternative: We could we could pre-create the directories so that an afterwards installed desktop will have the desktop icon. But I'd leave it as it is now.

Joulinar commented:
Indeed an option. Didn't think on that.

MichaIng committed:
v7.2

+ CHANGELOG | Kodi: Resolved an issue where an attempt was made during install to create a desktop entry, even if no desktop environment was installed

MichaIng commented:
I'll merge it like that. If someone ready misses a desktop icon for an afterwards installed desktop environment, we can rethink.
